### DISCUSSION TITLE
Branch for preview builds has changed from forked PR to commit id

### DISCUSSION THREAD

**Discussion by oskar-binary** [vercel/vercel#5271] [Help]
Hi,

We're facing an issue for our repo where the deployment of the preview links (starting from yesterday) are using the commit id as branch (in the build log) instead of the pull request.

This seems to cause the preview URLs to be re-created (on new different url) on each commit that is pushed - which in turn is causing issues for our QA process.

Any idea on what can be done to solve this?

See images here:
![image](https://user-images.githubusercontent.com/53067656/95271896-a5262800-0871-11eb-8171-b5e7c5b382fa.png)
![image](https://user-images.githubusercontent.com/53067656/95271902-ad7e6300-0871-11eb-97bd-5bab11a1c45d.png)

Example of a build with commit id as branch:
https://vercel.com/deriv/deriv-app/dg6ae27m6/binary.sx

**Comment by OskarAhl** [comment]
To reproduce:

1. Create a main repo
2. Fork repo
3. From fork submit PR to main repo

Tested with a personal dummy repo and account and same issue is happening: https://github.com/oskar-binary/lighthouse-aciton/pull/5
